pkg/views/workspace/list/view.go

```diff
@@ -204,6 +212,9 @@ func getProjectTableRowData(workspace serverapiclient.Workspace, project servera
 	if project.Repository != nil && project.Repository.Url != nil {
 		rowData.Repository = getRepositorySlugFromUrl(*project.Repository.Url, specifyGitProviders)
 	}
+	if project.Repository.Branch != nil {
```

Comment:
Same comment as above. Merge the two `if` blocks to run only if `project.Repository` is defined.